src/components/grid-aware/Footer/SubscriptionBlock/SubscriptionBlock.module.css

```diff
@@ -0,0 +1,77 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+/* Top Row */
+
+.subscriptionBlock {
+  background-color: var(--color-black);
+  display: flex;
+  grid-column: 1 / span 12;
+  justify-content: center;
+  padding: 35px;
+}
+
+.subscribeContainer {
+  align-items: baseline;
+  display: flex;
+}
+
+.subscribeMessage {
+  color: var(--color-white);
+  font: var(--font-subtitle);
+  margin-right: 20px;
+  width: 340px;
```

Comment:
I don't think we want to set a fixed width here. If the text changes, that will cause the effective spacing between the text and the form input to grow or shrink, and if we have too much text, it'll even wrap to the next line. Could you work out a value for `margin-right` that looks roughly the same with the current text but without a fixed width on the text box?

Reply:
margin-right: 97px;
  width: 263px;